website/project/model.py

```diff
@@ -3729,3 +3810,125 @@ def _on_reject(self, user, token):
             },
             auth=Auth(user),
         )
+
+
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
+
+    def _on_complete(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
+
+    def _on_reject(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
+
+
+class DraftRegistration(AddonModelMixin, StoredObject):
+
+    is_draft_registration = True
```

Comment:
Maintainability request: Please add a source code comment describing the conditions under which a DraftRegistration is not a Draft Registration.

(IIRC, this has to do with data being kept around after the draft is turned into a final registration)